staging/src/k8s.io/cli-runtime/pkg/manifest/manifest.go

```diff
@@ -0,0 +1,86 @@
+package kubernetes
+
+import (
+	"bytes"
+	"fmt"
+	"io"
+	"strings"
+
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apimachinery/pkg/util/yaml"
+	"k8s.io/client-go/kubernetes/scheme"
+)
+
+// NewRawExtensions decodes the provided manifest's bytes into "raw extension"
+// Kubernetes objects. These can then be passed to NewRawResource.
+func NewRawExtensions(manifest []byte) ([]runtime.RawExtension, error) {
```

Comment:
I think we mostly use Unstructured rather than RawExtension when the context isn't embedding an object.

Reply:
Thanks, I'll try to change this, to be honest these types are not very self-descriptive...